lib/core/wallet_connect/chain_service/solana/solana_chain_service.dart

```diff
@@ -72,7 +72,7 @@ class SolanaChainServiceImpl implements ChainService {
 
   @override
   List<String> getEvents() {
-    return [''];
+    return ['chainChanged', 'accountsChanged'];
```

Comment:
what are those?

Reply:
Some events that the Client listens to, similar to the eth_chain_service own also.